test_pegasus/dax/t1_processing.py

```diff
@@ -0,0 +1,142 @@
+import os
+
+from Pegasus.DAX3 import File, Job, Link
+from mappings import Inputs, T1Files, T1JobNames
+from qc_snapshots import QCSnapshots
+
+
+class T1Processing(object):
+    #TODO: how to define this in config?
+    subject = "TVB2PEG30"
```

Comment:
Could this be read from the properties file, together with FLAIR_FLAG, T2_FLAG, etc?